core/Cargo.toml

```diff
@@ -66,8 +65,6 @@ async-wasm-client = [
 
 [dev-dependencies]
 serde_json = "1.0"
-tokio = { version = "1.16", features = ["macros", "rt"] }
```

Comment:
sorry I meant that you could have `tokio = { version = "1.16", features = ["rt"] }`here and remove `tokio/rt` from `http-helpers`

Reply:
https://www.reddit.com/r/rust/comments/r5munx/comment/hmpgec2/?utm_source=share&utm_medium=web2x&context=3 Maybe I should give [cargo machete](https://blog.benj.me/2022/04/27/cargo-machete) a try.

Reply:
nah, I looked at your code and it's `dev-dependencies` to leaks in `cargo hack` pushing a fix soon :)

Reply:
Ok, I think `cargo machete` will give us less headache.

It looks like `dev-dependencies` doesn't work properly for us, basically we would have to `#[cfg(test)] pub use crate as _` for that to work.

> This lint is "allow" by default because it can provide false positives depending on how the build system is configured. For example, when using Cargo, a "package" consists of multiple crates (such as a library and a binary), but the dependencies are defined for the package as a whole. If there is a dependency that is only used in the binary, but not the library, then the lint will be incorrectly issued in the library.

:point_up: I guess that regards `tests` is treated as another binary and maybe it doesn't work